src/main/java/com/example/springBoot_hibernate_ferrocarriles/service/EquipoDeTraccionService.java

```diff
@@ -0,0 +1,45 @@
+package com.example.springBoot_hibernate_ferrocarriles.service;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.example.springBoot_hibernate_ferrocarriles.model.EquipoDeTraccion;
+import com.example.springBoot_hibernate_ferrocarriles.repository.EquipoDeTraccionRepository;
+
+@Service
+public class EquipoDeTraccionService {
+	@Autowired
+	private  EquipoDeTraccionRepository equipoDeTraccionRepository;
+	
+	public EquipoDeTraccionService() {
+		super();
+	}
+
+	public EquipoDeTraccionService(EquipoDeTraccionRepository equipoDeTraccionRepository) {
+		super();
+		this.equipoDeTraccionRepository = equipoDeTraccionRepository;
+	}
+	
+	public List<EquipoDeTraccion> getAllEquipoDeTraccion(){
```

Comment:
It is a good practice to log information about what the system is doing. A rule of thumb is to use loggers in all your Service class, because in general, they contain the dirty code executing the business rules. Example:
`
public List<EquipoDeTraccion> getAllEquipoDeTraccion(){
log.info("Retrieving all traction equipment");
......
}
`